utils/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+find_package( NetCDF REQUIRED COMPONENTS CXX)
+find_package(oops REQUIRED)
+find_package(atlas REQUIRED)
+find_package(soca REQUIRED)
+
+ecbuild_add_executable( TARGET gdas_socaincr2mom6.x
```

Comment:
How do you turn off building this when there is not 'OOPS' available? Like in GitHub CI?

Reply:
I eat my words (not your 🖱️). Looks good.